cypress/e2e/testScripts/VerifyLogin.cy.js

```diff
@@ -0,0 +1,35 @@
+/// <reference types="cypress"/>
+import { LoginPage } from "../../pages/LoginPage";
+
+describe("Login", function () {
+
+    Cypress.on('fail', (error, runnable) => {
+        if (error.message.includes('A fixture file could not be found')) {
+            console.log("Please check your file path or input file");
+            throw error;
+        }
+    });
+
+    const login = new LoginPage();
+
+    beforeEach(function () {
+        cy.log("Launch Swag Labs Application.");
+        cy.launchBrowser();
+
+        cy.log("Drive data form JSON file.");
+        cy.fixture('LoginTestData').then(function (LoginData) { this.LoginData = LoginData; })
+    })
+
+    afterEach(function () {
```

Comment:
Test steps should be properly organized.
1. Variable
2. beforeEach()
3. Tests
4. afterEach()